### PR TITLE
Django exercises

### PR DESCRIPTION
**Pull request by CharisseL**
@chortlehoort I needed you to take a look at what I have...is this in line with what we discussed to have them create a model for a blog post? Or is it too much or too little info?

### CODE REVIEW

File: 03-database-driven-applications/exercises/DBS_DJANGOTIME.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```
+$ python3 -m venv myvenv
+(replace myenv with name of your virtual environment. The above command will create a directory.)
+```
+
+Start working on that virtual environment.
+
+```
+$ workon myvenv
+You will know you are successfully using the virtual environment when you see (myvenv).
```
Comment (chortlehoort) [inline 03-database-driven-applications/exercises/DBS_DJANGOTIME.md]: Remove all instructions about creating a virtual environment

File: 03-database-driven-applications/exercises/DBS_DJANGOTIME.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+```
+$ django-admin.py startproject mysite .
+```
+
+Create database for the project. Make sure you run the following command in the directory where manage.py file is:
+
+```
+python manage.py migrate
+```
```
Comment (chortlehoort) [inline 03-database-driven-applications/exercises/DBS_DJANGOTIME.md]: Don't need to run migrate until **after** the model has been created and a migration has been created, so take this step and move it down to the very bottom

**Comment by CharisseL** [comment]
@chortlehoort for your review.

File: 03-database-driven-applications/exercises/DBS_DJANGOTIME.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+python manage.py runserver
+```
+
+Create the application (from the directory where manage.py is stored):
+
+```
+python manage.py startapp blog
+```
+
+After creating an application we also need to tell Django that it should use it. In mysite/settings/py, add a line containing 'blog' in the INSTALLED_APPS list.
```
Comment (chortlehoort) [inline 03-database-driven-applications/exercises/DBS_DJANGOTIME.md]: > Formatting

`mysite/settings.py`

**Comment by chortlehoort** [comment]
@CharisseL One formatting issue. Fix that and merge.  👍

**Pull request merged**